webapp-django/package.json

```diff
@@ -10,27 +10,27 @@
   "license": "MPL-2.0",
   "devDependencies": {
     "cssmin": "0.4.3",
-    "graceful-fs": "4.1.4",
-    "less": "2.7.1",
+    "graceful-fs": "4.1.11",
+    "less": "2.7.3",
     "nsp": "3.1.0",
-    "uglify-js": "2.6.1"
+    "uglify-js": "3.3.5"
   },
   "dependencies": {
     "Select2": "github:select2/select2#3.5.4",
-    "ace-builds": "1.2.8",
-    "d3": "3.5.17",
-    "d3-sankey": "0.4.2",
-    "filesize": "2.0.3",
-    "flatpickr": "4.0.3",
-    "font-awesome": "4.6.1",
+    "ace-builds": "1.2.9",
+    "d3": "4.12.2",
```

Comment:
That feels like a big jump. Did they change where the files ended up? Is that why we had to update the pipeline configuration?

Reply:
Yep.